curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/6443cf1cca1fbb02a8a6e46e.md

```diff
@@ -0,0 +1,292 @@
+---
+id: 6443cf1cca1fbb02a8a6e46e
+title: Step 31
+challengeType: 0
+dashedName: step-31
+---
+
+# --description--
+
+Create a constant named `daysAgo` that will store the number of days that have passed since the last post. You can do this by dividing `hoursAgo` by `24`, and then rounding down to the nearest whole number. Here is an example:
+
+```js
+const daysAgo = Math.floor(hoursAgo / 24); // 1 day = 24 hours
+```
```

Comment:
we should not supply the full answer here

```suggestion
Create a constant named `daysAgo` that will store the number of days that have passed since the last post. You can do this by dividing `hoursAgo` by `24`, and then rounding down to the nearest whole number. 

```